app/models/note.rb

```diff
@@ -15,4 +15,29 @@ class Note < ApplicationRecord
   has_one :utility, through: :user
   enum note_type: { review: 0, critique: 1 }
   validates :user_id, :title, :content, :note_type, presence: true
+  validate :validate_content_word_count
+
+  def word_count
+    content.split.size
+  end
```

Comment:
:fire: gracias por revisar las validaciones!

tip: Muchas veces lo que hacemos es usar el parámetro `&` para cortar un encadenamiento de métodos si en algún punto se devuelve `nil`

```suggestion
  def word_count
    content&.split.size
  end
```

Con ese ajuste no tendríamos que preocuparnos por tener la validación explicita en varios lados